src/components/ItemRecognition/ItemRecognition.js

```diff
@@ -82,6 +82,12 @@ class ItemRecognition extends Component {
         ) {
           await this.addTrainingImage(img.src, item.id);
           this.props.setPrediction(item.id, img.src);
+
+          const suggestions = [];
+          for (let i = 1; i < 4; i++) {
```

Comment:
maybe instead of a loop can use .slice to get the number you want then map to get ids?

Reply:
Is that more efficient?